pages/sites/[slug]/profile/bulk-codes/[method]/[id].tsx

```diff
@@ -75,32 +97,61 @@ export default function BulkCodeIssueCodesPage(): ReactElement {
     }
   }, [isReady, planetCashAccount]);
 
+  React.useEffect(() => {
+    if (router.isReady) {
+      setTenantConfig(pageProps.tenantConfig);
+    }
+  }, [router.isReady]);
+
   useEffect(() => {
     checkContext();
   }, [checkContext]);
 
-  return (
+  return pageProps.tenantConfig ? (
     <UserLayout>
       <Head>
         <title>{ready ? t('bulkCodesTitleStep3') : ''}</title>
       </Head>
       <BulkCodes step={BulkCodeSteps.ISSUE_CODES} />
     </UserLayout>
+  ) : (
+    <></>
   );
 }
 
-export const getStaticPaths: GetStaticPaths = async () => {
+export const getStaticPaths = async () => {
```

Comment:
Another point that can be dealt with in a later PR

1. Standardize the way `getStaticPaths` is called in all places - in some places it is currently an arrow function assigned to a `const getStaticPaths`, and other places a normal function `getStaticPaths`
2. Use the `GetStaticPaths` type from `next` to get warned about mistakes in return value

Reply:
@mohitb35

Reply:
Opened separate issue - https://github.com/Plant-for-the-Planet-org/planet-webapp/issues/1977